### PR TITLE
Coverity

### PR DESCRIPTION
**Pull request by akallabeth**
pull complete so far, will create another one with the remaining (low priority code cleanup) fixes

### CODE REVIEW

File: winpr/libwinpr/sspi/Negotiate/negotiate.c [hunk 1]
```diff
@@ -96,6 +96,7 @@ static const WinPrAsn1_OID negoex_OID = { 10, (BYTE*)"\x2b\x06\x01\x04\x01\x82\x
 
 #ifdef WITH_KRB5
 static const SecPkg SecPkgTable[] = {
+	{ KERBEROS_SSP_NAME, &KERBEROS_SecurityFunctionTableA, &KERBEROS_SecurityFunctionTableW },
```
Comment (mfleisz) [inline winpr/libwinpr/sspi/Negotiate/negotiate.c:99]: Looks like a duplicate

Comment (akallabeth) [reply winpr/libwinpr/sspi/Negotiate/negotiate.c:99]: @mfleisz yes, but look at the struct array below ;)

**Comment by chipitsine** [comment]
too many fixes ))
worth adding a badge to README

**Comment by akallabeth** [comment]
> too many fixes )) worth adding a badge to README

will do once we´re through with the reports.

**Review by mfleisz — APPROVED** [review]

**Pull request merged**